gradle.properties

```diff
@@ -28,7 +28,8 @@ projectRoot=${project.projectDir}
 
 org.gradle.daemon=false
 org.gradle.jvmargs=-Xmx1536m
-org.gradle.parallel=true
+# FIXME: Temporary disabled because of an issue in IntelliJ IDEA
+org.gradle.parallel=false
```

Comment:
is it on every IDEA? Or just some versions?
I'm asking if it's worth the performance penalty, or it could be set by each of us in our own gradle properties.

Reply:
Comment added:

```
# FIXME: Temporary disabled because of an issue in IntelliJ IDEA
# https://youtrack.jetbrains.com/issue/IDEA-323055/Gradle-with-GraalVM-buildtools-plugin-fails-to-import-on-2023.2-EAP-5
# The issue appeared probably at version 2023.1.4 and it is still valid in 2023.1.5
```

Reply:
I just think that this may impact negatively the build times for people using older IDEAs or not using IDEA (me for example) that's why maybe it's better to leave it enabled and communicate that they should disable the parallel if they run into the issue.

Reply:
Yes, it could... I can remove it and leave it on every user. Just for now with the current version is not possible to import the project properly. It is because of the Micronaut modules. Do you think it is better to leave true just with a comment?

Reply:
I think so, otherwise I fear we might forget to bring this back.
Maybe updating the `docs/ide-setup.md` would be fine too

Reply:
Updated